starport/services/networkbuilder/networkbuilder.go

```diff
@@ -456,6 +458,48 @@ func (b *Builder) StartChain(ctx context.Context, chainID string, flags []string
 	return g.Wait()
 }
 
+// Generate the genesis from the launch information in a temporary directory and return this directory
+func (b *Builder) GenerateTemporaryGenesis(ctx context.Context, chainID string, homeDir string, launchInfo spn.LaunchInformation) (string, error) {
+	tmpHome, err := ioutil.TempDir("", chainID+"*")
```

Comment:
I think there is no need for a chain id prefix in the temp dir.
```suggestion
	tmpHome, err := os.MkdirTemp("", "")
```

Also: https://golang.org/doc/go1.16#ioutil